packages/twenty-ui/src/theme/constants/BackgroundDark.ts

```diff
@@ -11,6 +11,7 @@ export const BACKGROUND_DARK = {
   secondary: GRAY_SCALE.gray80,
   tertiary: GRAY_SCALE.gray75,
   quaternary: GRAY_SCALE.gray70,
+  quinary: GRAY_SCALE.gray65,
```

Comment:
@Bonapara 
what do u think about this? it is used if that field is active or selected.
because if kept active card as quaternary, focused card would be tertiary and it is barely visible in dark mode.

Reply:
I agree, Quaternary is too dark. Let's stick to Tertiary (Focus = hover style) for now. I will work on the global focus across the app later.

Reply:
Okay! done